fem/pfespace.hpp

```diff
@@ -428,6 +428,33 @@ class ParFiniteElementSpace : public FiniteElementSpace
       old_dof_offsets.DeleteAll();
    }
 
+   /** @brief Return the true-dof to true-dof update operator corresponding to
+       the last Mesh::Operation. */
+   /** This method is similar to the GetUpdateOperator() methods, however the
+       returned operator works on T-dof (true-dof) vectors instead of L-dof
+       vectors (i.e. ParGridFunction-sized vectors).
+
+       @param[in,out]  T  On input, the type of @a T is used as input to request
+                          a given type of Operator::Type; currently only
+                          Operator::Hypre_ParCSR is supported. On output, @a T
+                          holds a pointer to the requested Operator and @a T
+                          owns the pointer. The returned Operator is constructed
+                          to be independent of the ParFiniteElementSpace, i.e.
+                          modifying or destroying the ParFiniteElementSpace will
+                          not invalidate the returned Operator. If an error is
+                          encountered (e.g. the requested type is not supported)
+                          the object @a T is cleared (stores the NULL pointer).
```

Comment:
Does empty `T` always mean that an error occurred? What happens when you call this on an unmodified `ParFiniteElementSpace`? Does it return an empty `T` or an identity operator?

Reply:
@tzanio If the FE space is unmodified, does this return an identity operator?

Reply:
It does not seem like that this method returns an identity operator for unmodified mesh currently, @tepperly. 
Regarding your question about checking whether the mesh is modified or not: `Mesh::GetSequence()` is the method to help figure that out.